master/static/migrations/20240910041102_alter-user-session-add-long-lived-token-info.tx.up.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TYPE token_type AS ENUM ('USER_SESSION', 'LONG_LIVED_TOKEN');
+ALTER TABLE user_sessions ADD COLUMN created_at TIMESTAMP DEFAULT NULL;
+ALTER TABLE user_sessions ADD COLUMN token_type token_type NOT NULL DEFAULT 'USER_SESSION';
+ALTER TABLE user_sessions ADD COLUMN token_description TEXT DEFAULT NULL;
```

Comment:
it might be worth adding a conditional unique index to this table to enforce that 1:1 relationship between users & long lived tokens; something kinda like:
```
create unique index on user_session (user_id) where token_type = 'LONG_LIVED_TOKEN';
```

Reply:
Done